Keyed/Dialogs_Various.xml

```diff
@@ -263,11 +263,11 @@
   <!-- EN: Minimum skills: -->
   <MinimumSkills>Min. poziom umiejętności:</MinimumSkills>
   <!-- EN: Requires {0} {1} -->
-  <BillRequires>Wymaga {0} {1}</BillRequires>
+  <BillRequires>Wymaga: {1} w liczbie {0} </BillRequires>
   <!-- EN: Requires {0} nutrition -->
   <BillRequiresNutrition>Wymaga {0} wartości odżywczej</BillRequiresNutrition>
   <!-- EN: Note: Small-volume ingredients like gold require {0}x as much. -->
-  <BillRequiresMayVary>Uwaga: niewielkich składników takich jak złoto potrzeba {0}x więcej.</BillRequiresMayVary>
+  <BillRequiresMayVary>Uwaga: Składników o małej objętości jak złoto potrzeba {0}x więcej.</BillRequiresMayVary>
```

Comment:
nie chodzi tu przypadkiem o rzadkie składniki ??

Reply:
<ThisIsSmallVolume>
To niewielki składnik. Oznacza to, że każda jego jednostka odpowiada zaledwie 1/{0} składnika normalnej wielkości. Dlatego też potrzebujesz go {0}x więcej niż innych składników wykorzystywanych do tych samych celów.
</ThisIsSmallVolume>